api/db/apd.js

```diff
@@ -104,8 +104,6 @@ module.exports = () => ({
 
     static: {
       updateableFields: [
-        'status',
-        'period',
```

Comment:
I don't even know what `period` refers to, but `status` shouldn't be settable directly.  It should be inferred by other actions (e.g., submitting an APD, sending it back to a state for responses, etc.).

Reply:
i think period was originally intended to encompass the year range for an APD (i.e., 2018-2020)